### PR TITLE
feat: Add team member component modal

### PR DESCRIPTION
**Pull request by faraz16iqbal**

### CODE REVIEW

**Comment by faraz16iqbal** [comment]
Screenshot from 2021-02-10 20-04-25

**Comment by iamareebjamal** [comment]
Add a margin above interests and resolve conflicts

**Comment by faraz16iqbal** [comment]
Added margin and resolved conflicts

File: pages/team.vue [hunk 1]
```diff
@@ -71,13 +153,33 @@ import ResizeImg from '@/components/widgets/ResizeImg.vue'
...
+        name: '',
+        profile_info: {
+          about: '',
+          batch: '',
+          cv: '',
+          interests: []
+        },
+        thumbnail: '',
+        uid: ''
+      }
```
Comment (iamareebjamal) [inline pages/team.vue:169]: Why these? Why not null?

Comment (faraz16iqbal) [reply pages/team.vue:169]: Was using null initially but was told to review the code from the projects section.

Comment (faraz16iqbal) [reply pages/team.vue:169]: Should I change it to null?

Comment (iamareebjamal) [reply pages/team.vue:169]: Yes

File: pages/team.vue [hunk 2]
```diff
@@ -71,13 +153,28 @@ import ResizeImg from '@/components/widgets/ResizeImg.vue'
...
   components: { PageLoader, ResizeImg },
+  data() {
+    return {
+      dialog: false,
+      selectedMember: {
+        name: null,
+        profile_info: null,
+        thumbnail: null,
+        uid: null
+      }
```
Comment (iamareebjamal) [inline pages/team.vue:164]: Why this and not null?

Comment (faraz16iqbal) [reply pages/team.vue:164]: Can't set selectMember to null as the uid is needed for conditional rendering. Instead can set it to an empty object

**Comment by iamareebjamal** [comment]
Why using uid for conditional rendering?

**Comment by faraz16iqbal** [comment]
The dialog box was just for the members, not the faculty. Faculty members don't have a UID property

**Comment by iamareebjamal** [comment]
Still no margin above interests

**Comment by faraz16iqbal** [comment]
Screenshot from 2021-02-10 23-30-37

**Comment by faraz16iqbal** [comment]
More?

**Comment by iamareebjamal** [comment]
107551246-07b07b80-6bf8-11eb-9de9-1214aec06aff

If by "More?", you mean more than zero, then yes

**Comment by faraz16iqbal** [comment]
Screenshot from 2021-02-10 23-49-18

Is this better?

**Comment by iamareebjamal** [comment]
Not really

**Comment by iamareebjamal** [comment]
Still nothing

**Comment by faraz16iqbal** [comment]
Screenshot from 2021-02-14 22-47-10

**Comment by iamareebjamal** [comment]
Finally

**Comment by faraz16iqbal** [comment]
Should I add anything else?

**Comment by iamareebjamal** [comment]
I can only see the change in screenshot, not on the site

**Comment by faraz16iqbal** [comment]
Just pushed the changes

**Comment by faraz16iqbal** [comment]
Is it working now?

**Pull request merged**